.github/workflows/test.yml

```
@@ -8,7 +8,13 @@ on:
 
 jobs:
   test:
-    runs-on: "ubuntu-latest"
+    strategy:
+      fail-fast: false
+      matrix:
+        runs-on:
+          - "ubuntu-latest"
+          - "windows-latest"
```

Good idea, even if we don't do too much fancy stuff with file paths and Node.js should already have everything covered, it's better to be sure.

Let's add macOS as well so we are "complete" for the most used OS.

```suggestion
          - "windows-latest"
          - "macos-latest"
```